### PR TITLE
Release v1.2.0

### PR DESCRIPTION
**Pull request by jimdickinson**

### CODE REVIEW

File: buildsettings.yaml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 version:
   major: 1
-  minor: 1
-  patch: 1
-  prerelease: "snapshot"
+  minor: 2
+  patch: 0
+  prerelease: "rc1"
```
Comment (johntrimble) [inline buildsettings.yaml:5]: This is intentionally a release candidate, right?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,22 @@
...
+* Update to Operator SDK v0.17.0 [#78](https://github.com/datastax/cass-operator/commit/ac882984b78d9eb9e6b624ba0cfc11697ddfb3d2)
+* Update Cassandra images to include metric-collector-for-apache-cassandra (MCAC) [#81](https://github.com/datastax/cass-operator/commit/4196f1173e73571985789e087971677f28c88d09)
+* Run data cleanup after scaling up a datacenter [#80](https://github.com/datastax/cass-operator/commit/4a71be42b64c3c3bc211127d2cc80af3b69aa8e5)
+* Requeue after the last node has its node-state label set to Started during cluster creation [#77](https://github.com/datastax/cass-operator/commit/b96bfd77775b5ba909bd9172834b4a56ef15c319)
+* Remove delete verb from validating webhook [#75](https://github.com/datastax/cass-operator/commit/5ae9bee52608be3d2cd915e042b2424453ac531e)
+* Add conditions to CassandraDatacenter status [#50](https://github.com/datastax/cass-operator/commit/8d77647ec7bfaddec7e0e616d47b1e2edb5a0495)
+* Better support and safeguards for adding racks to a datacenter [#59](https://github.com/datastax/cass-operator/commit/0bffa2e8d084ac675e3e4e69da58c7546a285596)
+
```
Comment (johntrimble) [inline CHANGELOG.md:19]: Are you generating this list by hand or are you using a script for it?

File: buildsettings.yaml (hunk 1 above)
Comment (jimdickinson) [reply buildsettings.yaml:5]: right now, yeah, will switch it to `release` before merge

File: CHANGELOG.md (hunk 2 above)
Comment (jimdickinson) [reply CHANGELOG.md:19]: it was `git log --pretty=oneline` + some regexp to reorder the line, and I made some messages nicer

**Pull request merged**